acceptance/tests/config-entries/config_entries_namespaces_test.go

```diff
@@ -90,6 +90,10 @@ func TestControllerNamespaces(t *testing.T) {
 
 				"global.acls.manageSystemACLs": strconv.FormatBool(c.secure),
 				"global.tls.enabled":           strconv.FormatBool(c.secure),
+
```

Comment:
this test exercises the terminating gateway path and those don't function correctly without this in your helm chart